projects/sass-lang.com/sassc/package.yml

```diff
@@ -0,0 +1,26 @@
+distributable:
+  url: https://github.com/sass/sassc/archive/refs/tags/{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: sass/sassc
+dependencies:
+  sass-lang.com/libsass: '*'
```

Comment:
Sorry. We do not permit `'*'` dependencies any more.

Reply:
Okay, got it 👌🏻

Reply:
Am I correct in understanding that we should not use `'*'` in runtime dependencies? In the build section we can use this as before